Solutions/Menes.Abstractions/Menes/Internal/OpenApiExternalServices.cs

```diff
@@ -65,7 +65,7 @@ public Uri ResolveUrl<TService>(string operationId, params (string Name, object?
 
             ResolvedOperationRequestInfo relativeUrl = providerAndKey.Provider.GetResolvedOperationRequestInfo(operationId, parameters);
 
-            string baseUrl = this.configuration[providerAndKey.ConfigKey];
+            string baseUrl = this.configuration[providerAndKey.ConfigKey]!;
```

Comment:
Is simply suppressing the nullability warning the right thing to do here? Arguably the compiler has spotted a bug here: we don't handle the absence of required configuration correctly. Instead of `!`, it would be better to use `?? throw new InvalidOperationException($"Expected configuration {providerAndKey.ConfigKey} was not found")`. (Or if we have some more specific 'your config is bad' type exception, that would be even better.